lib/danger/dangerfile.rb

```diff
@@ -36,12 +36,16 @@ def print_known_info
         rows << [key.to_s, value]
       end
 
+      pr_copy = pr_body[0..80]
+      pr_copy += "..." if pr_body.length == 81
+      rows << ["PR Body", pr_copy]
+
       rows << ["---", "---"]
       rows << ["SCM", env.scm.class]
       rows << ["Source", env.ci_source.class]
       rows << ["Requests", env.request_source.class]
-      rows << ["Base Commit", env.request_source.base_commit]
-      rows << ["Head Commit", env.request_source.head_commit]
+      rows << ["Base Commit", env.scm.exec("--no-pager log #{env.danger_base_branch} -n1")]
+      rows << ["Head Commit", env.scm.exec("--no-pager log #{env.danger_head_branch} -n1")]
```

Comment:
shouldn't these be methods on `scm` to make it easier to add other sources later?